lib/absinthe/blueprint/document/operation.ex

```diff
@@ -18,6 +18,7 @@ defmodule Absinthe.Blueprint.Document.Operation do
     # Populated by phases
     flags: %{},
     schema_node: nil,
+    complexity: nil,
```

Comment:
Would it make sense to initialize this to the value of the default complexity?

Doing so would simplify a couple of function clauses in the phase.